util/packaging/homebrew/readme.md

```diff
@@ -1,16 +1,57 @@
-This folder contains both the current shipping homebrew formula `chapel-release.rb`, 
+
+This folder contains a copy of the shipping homebrew formula `chapel-release.rb`, 
 and the modified formula that would be used in the next chapel release as 
-`chapel-main.rb`.  To test local homebrew builds, the following items are 
-needed: 
+`chapel-main.rb`.  These are stored here to allow the Chapel team test upcoming changes without 
+impacting the Homebrew core project formula: (https://github.com/Homebrew/homebrew-core/blob/master/Formula/chapel.rb).   
+## Tracking changes
+The homebrew team will update formulas under some circumstances and this mostly when a formula
+has not specified a version for a dependency.  For example, the `chapel.rb` formula refers to the 
+most recent version of `llvm` via this line `depends_on "llvm"`.  If the `llvm` project updates their 
+formula, the homebrew team will modify dependencies by specifying a version such as `depends_on "llvm@13"`.
+They will also indicate the formula changed via the addtion of a revision value such as `revision 1` as seen
+in the 1.25.1 update:  https://github.com/Homebrew/homebrew-core/blob/c63259777f760d8cebdce758c3e91778e58b55a5/Formula/chapel.rb. 
+
+## How to release the formula 
+The homebrew docs explain how to issue a pull request here:  
+https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request.  Before issuing the request itself, 
+the following commands should be run before submitting the PR: 
+```
+brew tests
+brew install --build-from-source <CHANGED_FORMULA>
+brew test <CHANGED_FORMULA>
+brew audit --strict --online <CHANGED_FORMULA>`
+```
+
+These steps confirm that the formula will build within your local homebrew environment and 
+that the formula itself is properly formatted.  No trailing whitespace or extraneous characters
+thhat would cause the homebrew CI process to fail. 
+
+### Linuxbrew testing 
+The Ubuntu CI is run on a docker container per the instructions on their contribution documentation: 
+https://github.com/Homebrew/homebrew-core/blob/5e9db7dda1d327de4d2cccfc6976116175e0c72a/CONTRIBUTING.md
```

Comment:
I would imagine that https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md would be a better link here

Reply:
Updated the link - not sure why I did not pick up the main line version of the file.